### DISCUSSION TITLE
Welcome to SpotifyKaraoke Discussions!

### DISCUSSION THREAD

**Discussion by gedrick** [gedrick/SpotifyKaraoke#17] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by assadtony** [comment]
Hey! This is an awesome idea! I see that the domain is no longer available. Did you find some challenges or something happened that made this not workable anymore?
